doc/achievements.md

```diff
@@ -0,0 +1,160 @@
+<!--
+//===== rAthena Documentation ================================
+//= Achievement Database Structure
+//===== By: ==================================================
+//= rAthena Dev Team
+//===== Last Updated: ========================================
+//= 20200220
+//===== Description: =========================================
+//= Explanation of the achievements_db.yml file and structure.
+//============================================================
+-->
+
+# Achievement Database Structure
+
+```yml
+- Id: 99                  
+   Group: Baby            
+   Name: Example Achiev
+   Targets:            
+     - Id: 0
+       Mob: XM_CELINE_KIMI
+       Count: 1
+   Condition: " BaseLevel >= 99 "
+   Map: prontera
+   Dependents:
+     - Id: 100
+   Rewards:
+     Item: Shabby_Purse
+     Amount: 10            Amount of item. (Default: 1)
+     Script: " specialeffect2 EF_BLESSING; sc_start SC_BLESSING,30000,10; "
+     TitleId: 1000
+   Score: 10
+```
+
+### Id
+Unique achievement ID.
+
+### Group
+> Achievement group type.  
```

Comment:
I think that's because I missed the quote on the item above, so they look like this but we could remove the quotes
<img width="488" alt="Screenshot 2023-03-30 at 14 57 34" src="https://user-images.githubusercontent.com/13068064/228859899-c996a26d-a5e8-4483-a021-2cc43f4e5aa4.png">